docs/developer_guide/pull-request.md

```diff
@@ -0,0 +1,34 @@
+# Creating a Pull Request
+
+Once you've made your changes to the MedModels codebase and are ready to share them, you'll submit a pull request (PR). Here's a guide to walk you through the process:
+
+**1. Push your Local Branch:**
+
+- Commit your changes locally using clear and concise commit messages following the Conventional Commits standard ([https://www.conventionalcommits.org/en/v1.0.0/](https://www.conventionalcommits.org/en/v1.0.0/)).
```

Comment:
I would say standard guidelines instead of just standard

Reply:
using standard as noun instead of adjective should be fine, no?

Reply:
Yeah, could be. I just like it better the other way but it s also working